package/src/components/Link/v1/Link.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { preventAccidentalDoubleClick } from "../../../utils";
+
+class Link extends Component {
+  static propTypes = {
+    /**
+     * The contents of the link, such as text, icons, or any combination of React and HTML components
+     */
+    children: PropTypes.node.isRequired,
+    /**
+     * The URL the link should navigate to
+     */
+    href: PropTypes.string.isRequired,
+    /**
+     * Called with a single event parameter when a user clicks the link
+     */
+    onClick: PropTypes.func.isRequired
+  };
+
+  static defaultProps = {
+    onClick(event) {
```

Comment:
Since you have the onClick func set as `isRequired` I don't think you'd need this default. Or I would probably make the `onClick` a no-op default something like `defaultProps = { onClick() {} }` and make the `onClick` func optional.

We'd also like to avoid using DOM object within components since these could render on the server. If you have to access one of those objects wrap it in a conditional like `if (typeof window !== "undefined") { ... }`